pkgs/top-level/python-packages.nix

```
@@ -6020,6 +6020,8 @@ in {
 
   yarl = callPackage ../development/python-modules/yarl { };
 
+  solo-python = disabledIf (! pythonAtLeast "3.6") (callPackage ../development/python-modules/solo-python { });
```

please move disable logic into expression
```suggestion
  solo-python = callPackage ../development/python-modules/solo-python { };
```

---

Done